packages/create-instance/create-instance.js

```diff
@@ -55,7 +55,7 @@ export default function createInstance(
   const stubComponentsObject = createStubsFromStubsObject(
     componentOptions.components,
     // $FlowIgnore
-    options.stubs,
+    { ...options.stubs, transition: true },
```

Comment:
Shouldnt we add this to the default stubs in options? Wont it be a bit cleaner?

Reply:
Yeah this might not be the best place to add this - which line/file do you think we should add this?

Reply:
Didnt we have a default global `stubs` object?

Reply:
we do - updated the PR, good suggestion